### DISCUSSION TITLE
EQ5 reflections

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/library#245] [General]
- Reword [Activity 1.5.2](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#determine-quadratic-equation) to indicate that there is more than one answer.

- [Activity 1.5.4(a)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#applying-zero-product-property-2) - I'd suggest changing part (d) so that it's not equal to 0. That way they aren't just looking for factored pieces, but factored pieces that are set equal to 0. Also, changing answer choice B to match the question they are doing in part b would be good. Some of my students were tripped up in part b because the new quadratic was so close to the one in part a answer choice B.

- My students did not remember how to factor for [Activity 1.5.6](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#solve-unfactored-equation). They mostly did guess and check to narrow down to the right answer for part (a). Which is fine here, I guess. So maybe add factoring when a isn't 1 to the RAT. Or switch this one so that a=1. Then they can practice the idea without factoring holding them back as much. [Activity 1.5.7(c)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#solve-quadratic-equations-practice-4) also gave them trouble. We talked about how to factor it, but usually I would say that if they don't see how to factor it fairly quickly to just use quadratic formula. But we weren't there yet.

- In [Activity 1.5.7](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#solve-quadratic-equations-practice), it would be nice to have one where a=1 and it's set up in standard form, maybe between b and c.

**Comment by AbbyANoble** [comment]
[Definition 1.5.18](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#def-quadratic-formula) 
should probably be split into two definitions. A first one to define standard form as when a quadratic is written as $ax^2+bx+c=0$. Then a second defining the quadratic formula. This is because the next activities ask for standard form but we had not yet mentioned it.

Speaking of definitions, we might want to adjust [Definition 1.5.1](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#def-quadratic-equation) to say that a quadratic equation is an equation that can be written in the form $ax^2+bx+c=0$ where $a$, $b$, and $c$ are real numbers, $a\not = 0$. As written it sounds like it has to be written in that form to be quadratic.

At the end of the section, it may be worthwhile to (one day) add an activity where the quadratic formula is developed from completing the square.

**Comment by AbbyANoble** [comment]
In [Activity 1.5.20(c)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#quadratic-formula-square-root-4) the answer choice does not simplify underneath the root. $\sqrt{153}$ stays as that rather than $3\sqrt{17}$. But then in [Activity 1.5.21(c)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#quadratic-formula-imaginary-4), the answers are simplified. [Activity 1.5.22(b)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#quadratic-formula-practice-3) also simplified. When are we simplifying and when are we not? 

Also in [Activity 1.5.21(c)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#quadratic-formula-imaginary-4), a good wrong answer may be $3 \pm 2i\sqrt{3}$. This comes from if they just simplify the first term of the numerator with the denominator instead of the whole numerator.

[Activity 1.5.22(d)](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#quadratic-formula-practice-5) is the same as [Activity 1.5.21](https://library.tbil.org/2024/precalculus/instructor/EQ5.html#quadratic-formula-imaginary), just written differently. Was this on purpose or do we want another example there?

**Comment by siwelwerd** [comment]
See #565
